core/mod/CoreNEURONArtificialCell.mod

```diff
@@ -0,0 +1,18 @@
+COMMENT
```

Comment:
For my understanding only, we have in place a way to create fake cells for CoreNeuron. Is this replacing it? If so, do you think we will be able to remove that Fake Cell workaround?

Reply:
Unfortunately we can't replace the workaround since we need to have a cell (`ARTIFICIAL_CELL`) in every process. Then `ARTIFICIAL_CELL` needs to have a `NET_RECEIVE` and a `NET_RECEIVE` has to have a `net_event` so we still need this code